libcxx/include/__ranges/zip_view.h

```diff
@@ -251,6 +252,10 @@ class zip_view<_Views...>::__iterator : public __zip_view_iterator_category_base
 
   friend class zip_view<_Views...>;
 
+  using __is_zip_view_iterator _LIBCPP_NODEBUG = true_type;
```

Comment:
```suggestion
  static constexpr bool __is_zip_view_iterator = true;
```

Slight simplification. This is what we do for `join_view::iterator` and segmented iterator traits.